emails/file-expiry.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+    Hr,
+    Link,
+    Section,
+    Text
+} from "@react-email/components";
+import { z } from "zod";
+import { WATcloudEmail } from "./_common/watcloud-email";
+
+const EmailProps = z.object({
+    name: z.string(),
+    paths: z.array(z.string()),
+    daysForExpiry: z.string(),
+    deletionDate: z.string()
+});
+
+type EmailProps = z.infer<typeof EmailProps>;
+
+const Email = (props: EmailProps) => {
+    const { name, paths, daysForExpiry, deletionDate} = EmailProps.parse(props);
+
+    const previewText = `You have expired files in the WATO drives`;
+
+    const expiredPaths = (
+        <div>
+            <ul style={{ fontSize: "14px", lineHeight: "24px" }}>
+                {paths.map(path => (
+                    <li key={path}>{path}</li>
+                ))}
+            </ul>
+        </div>
+    );
+
+    return (
+        <WATcloudEmail previewText={previewText}>
+            <Text>Hi {name},</Text>
+            <Text>
+                In an ongoing effort to keep WATcloud's file storage efficient and maintain optimal performance, we routinely clean up specific directories that haven't been accessed in the past {daysForExpiry} days.            
+            </Text>
+            <Hr />
+            <Section>
+                <Text>The following paths on each respective machine are expired:</Text>
+                {expiredPaths}
+            </Section>
+            <Hr />
+            <Text style={{ fontWeight: 'bold' }}>
+                If you do not access files in any of these directories, then they will be automatically deleted on {deletionDate}.            </Text>
```

Comment:
Formatting issue here too. Perhaps run a formatter on this file to fix all the formatting issues?